src/internal/utils/exec.go

```diff
@@ -17,10 +18,10 @@ const colorCyan = "\x1b[36;1m"
 const colorWhite = "\x1b[37;1m"
 
 //nolint
-func ExecCommand(showLogs bool, envVariables []string, commandName string, args ...string) (string, error) {
+func ExecCommandWithContext(ctx context.Context, showLogs bool, commandName string, args ...string) (string, error) {
 	if showLogs {
 		fmt.Println()
-		fmt.Printf("%s", colorGreen)
+		fmt.Printf("  %s", colorGreen)
```

Comment:
Was this space intentional?

Reply:
yes it was, just aligning that with other logs in the message pkg.